### PR TITLE
[nic] README VLAN add an additional explanation

### PR DESCRIPTION
**Pull request by hmeAtos**
Hi guys,

I added some explanations on the VLAN which were not clear to some users.

Regards,
Hamid

### CODE REVIEW

File: roles/core/nic/readme.rst [hunk 1]
```diff
@@ -51,7 +51,13 @@ To configure an LACP bonding, specify slave interfaces, and then create the bond
       type: bond-slave
       master: bond0
 
-To configure a vlan, simply set vlan to true:
+To configure a VLAN, simply set VLAN to true.
```
Comment (btravouillon) [inline roles/core/nic/readme.rst:54]: The parameter is `vlan` (lowercase): ... set vlan to true.

File: roles/core/nic/readme.rst [hunk 2]
```diff
@@ -51,7 +51,13 @@ To configure an LACP bonding, specify slave interfaces, and then create the bond
...
+
+In this example we want to use a dedicated subnet for VLAN 100, we have created a net-100 network (corresponding to VLAN 100).
+As a reminder, the creation of a new network requires a new entry in the 
+/etc/bluebanquise/inventory/group_vars/all/general_settings/network.yml file
```
[the remark this answers is not in the document]
Comment (hmeAtos) [reply roles/core/nic/readme.rst]: Its ok @btravouillon 
thanks

File: roles/core/nic/readme.rst (hunk 1 above)
Comment (hmeAtos) [reply roles/core/nic/readme.rst]: good
thank you

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

File: roles/core/nic/readme.rst [hunk 3]
```diff
@@ -13,6 +13,9 @@ This role provide network configuration based on system files (ifcfg files for R
 
 User need to restart network or interfaces one by one for changes to take effect after role execution.
 
+As a reminder, the creation of a new network requires a new entry in the 
+``/etc/bluebanquise/inventory/group_vars/all/general_settings/network.yml`` file
```
Comment (oxedions) [inline roles/core/nic/readme.rst:17]: Missing dot at end of the line :-)

File: roles/core/nic/readme.rst [hunk 4]
```diff
@@ -51,7 +54,11 @@ To configure an LACP bonding, specify slave interfaces, and then create the bond
       type: bond-slave
       master: bond0
 
-To configure a vlan, simply set vlan to true:
+To configure a VLAN, simply set vlan to true.
+
+In this example we want to use a dedicated subnet for VLAN 100, we have created a net-100 network (corresponding to VLAN 100).
```
Comment (oxedions) [inline roles/core/nic/readme.rst:59]: Can you split line in 2 ? So users can easily read in a shell.

**Review by oxedions — COMMENTED** [review]
Dear @hmeAtos 

Thanks a lot for this PR 😊 

2 small comments, not major but if you can fix them, then I will merge it.

File: roles/core/nic/readme.rst (hunk 4 above)
Comment (hmeAtos) [reply roles/core/nic/readme.rst]: Done

File: roles/core/nic/readme.rst (hunk 3 above)
Comment (hmeAtos) [reply roles/core/nic/readme.rst]: Done

**Comment by oxedions** [comment]
Thanks a lot !
Merging.

**Pull request merged**